flight/UAVObjects/inc/uavobjecttemplate.h

```diff
@@ -39,6 +39,8 @@
 #define $(NAMEUC)_H
 
 #include "pios_queue.h"
+#include "uavover.h"
```

Comment:
Not to beat a dead horse, but this could be parsed like "UAV over", when we want people to understand "UAVO Version".